src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -228,12 +254,14 @@ public static IEnumerable<object[]> GetFileNameWithoutExtension_TestData()
         public static void GetFileNameWithoutExtension(string path, string expected)
         {
             Assert.Equal(expected, Path.GetFileNameWithoutExtension(path));
+            Assert.Equal(expected ?? string.Empty, new string(Path.GetFileNameWithoutExtension(path != null ? path.AsReadOnlySpan() : new ReadOnlySpan<char>())));
         }
 
         [Fact]
         public static void GetPathRoot_NullReturnsNull()
         {
             Assert.Null(Path.GetPathRoot(null));
+            Assert.Equal(string.Empty, new string(Path.GetPathRoot(new ReadOnlySpan<char>())));
```

Comment:
Assuming there is a test for empty returns empty, this test has no meaning for span, remove the line